recipes/aws-noaa-fcdr-micro-brit-tmp/recipe.py

```diff
@@ -0,0 +1,63 @@
+import boto3
+
+from pangeo_forge_recipes.patterns import pattern_from_file_sequence
+
+# from pangeo_forge_recipes.recipes.reference_hdf_zarr import HDFReferenceRecipe
+
+client = boto3.client('s3')
+bucket_name = 'noaa-cdr-microwave-brit-temp-pds'
+
+paginator = client.get_paginator('list_objects_v2')
+
+# 'list_objects_v2': Returns some or all (up to 1000) of the objects in a bucket
+# this could be a limiting factor
+
+response_iterator = paginator.paginate(
+    Bucket=bucket_name,
+    Prefix='data',
+)
+
+# sensor_list = ['N15', 'N16', 'N17', 'N18', 'AQUA', 'M02']
+
+# psuedo code:
+# iterate through sensor list, if sensor (e.g., N15) is in file_list
+# iterate through year in years and create separate archive for each sensor
+# want to ignore files that don't include GRID in filename
+# for now group ascending and descending together
+
+for result in response_iterator:
+    contents = result.get('Contents', [])
+    # can implement filter here
+    for f in contents:
+        key = f['Key']
+        # print(key)
+        # look at only gridded files
+        if '-GRID_' in key:
+            # to iterate through yearly subfolders = key.split('/')[1]
+            for key.split('/')[1] in key:
+                if '_N15_' in key:
+                    pattern_N15 = pattern_from_file_sequence(key, 'time')
```

Comment:
I think it makes sense to set this one aside for now given the complexity, but eventually I think if each of the sensors are not aligned on the same grid, then these conditionals can be in separate recipes. If datasets are in different files but on the same grid, then they can be merged and then I think two patterns from file sequence make sense but maybe with a merge step. this doc page indicates tha twhile concat dim is used for concatenating files of the same variable, merge dim is typically used for combining multiple variables: https://pangeo-forge.readthedocs.io/en/latest/pangeo_forge_recipes/api_reference.html